### ISSUE TITLE
MatplotlibDeprecationWarning using matplotlib 3.3.1

### ISSUE THREAD

**Issue by altmas5** [molu8bits/modsecurity-parser#8]
If matplotlib version is 3.3.1 you get the following deprecation warning:

```
modsecurity-parser.py:454: MatplotlibDeprecationWarning: normalize=None does not normalize if the sum is less than 1 but this behavior is deprecated since 3.3 until two minor releases later. After the deprecation period the default value will be normalize=True. To prevent normalization pass normalize=False 
  patches, texts, autotexts = plt.pie(intercepted_cnt_top10.values(), autopct='%1.1f%%', shadow=True, startangle=90, radius=1.0)
```

I've read that you're using an older version of that python lib, but in case you're interested in supporting newer versions you can update your code

**Comment by molu8bits** [comment]
thanks for your contribution, looks good, Will review, build and merge soon your PR

**Comment by molu8bits** [comment]
Seems the normalize must be True
Otherwise for bigger files there is an exception like below:
`inputFileName : modsec3/modsec_audit.log
---------- modsec_audit events processed: 577    ----------
---------- modsec_audit events skipped by INCLUDED/EXCLUDED options or INVALID : 0 ----------
Traceback (most recent call last):
  File "modsecurity-parser3.py", line 689, in <module>
    outputWithGraphs = modsecViewGraphs(modsec_entries)
  File "modsecurity-parser3.py", line 454, in modsecViewGraphs
    patches, texts, autotexts = plt.pie(intercepted_cnt_top10.values(), autopct='%1.1f%%', shadow=True, startangle=90, radius=1.0, normalize=False)
  File "/usr/local/lib/python3.8/dist-packages/matplotlib/pyplot.py", line 2827, in pie
    return gca().pie(
  File "/usr/local/lib/python3.8/dist-packages/matplotlib/__init__.py", line 1438, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/matplotlib/axes/_axes.py", line 3018, in pie
    raise ValueError('Cannot plot an unnormalized pie with sum(x) > 1')
ValueError: Cannot plot an unnormalized pie with sum(x) > 1`

and charts are not created. When normalize=True everything works without warning and charts are generated. 
I checked that for modsec_audit.log traditional and json version. Also modsecurity3 logs behave in the same way.

I have added that with the newest PR which is already merged.
